Ensure cuda_primary_ctx test is run on multigpu CI

lw opened the pull request:
Summary: It requires 2 GPUs, but it wasn't added to the list of tests running on multigpu jobs.

Test Plan: Look at CI

Differential Revision: D38616784

lw commented:
See https://github.com/pytorch/pytorch/runs/7787082400

malfet reviewed (approved):
LGTM, though probably we want to rename `distributed` to multigpu job

lw commented:
@pytorchbot merge

lw commented:
@pytorchbot rebase -s

lw commented:
@pytorchbot merge -f "Only one impacted job, which passes: https://github.com/pytorch/pytorch/runs/7896669045"

lw committed:
Ensure cuda_primary_ctx test is run on multigpu CI (#83252)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/83252

It requires 2 GPUs, but it wasn't added to the list of tests running on multigpu jobs.

Test Plan: Look at CI

Reviewed By: malfet

Differential Revision: D38616784

fbshipit-source-id: ffad31b8ec7c0f1908608fd01e6c98597f4361fd

lw commented:
@pytorchbot merge -f "Same as before"
